openshift/template.yaml

```diff
@@ -397,6 +397,8 @@ objects:
       /opt/kafka/bin/kafka-acls.sh --authorizer-properties zookeeper.connect=${ZOOKEEPER} --add --allow-principal "User:${READ_USERNAME}" --operation Read --group "${KAFKA_GROUP_ID}" --topic "${KAFKA_TOPIC}" ${ZK_TLS}
       # setup write user
       /opt/kafka/bin/kafka-acls.sh --authorizer-properties zookeeper.connect=${ZOOKEEPER} --add --allow-principal "User:${WRITE_USERNAME}" --operation Write --topic "${KAFKA_TOPIC}" ${ZK_TLS}
+      # setup Mirrormaker user
+      /opt/kafka/bin/kafka-acls.sh --authorizer-properties zookeeper.connect=${ZOOKEEPER} --add --allow-principal "User:${MIRRORMAKER_USERNAME}" --operation Read --operation Write --topic "${MIRRORMAKER_TOPIC}" ${ZK_TLS}
```

Comment:
I am not sure about what ACLs a MirrorMaker2 (contrary to MirrorMaker, which was really simple, too) might require. Especially as it also can replicate topic settings, etc. I suppose the most basic usage would suffice? Or would it simply work?

Reply:
I think this looks good! MirrorMaker may try to create a few extra topics, so we may have to expand the access it has to topics, but I think starting with this minimal set and then expanding if we get errors is the right way to go.

Reply:
@wcmitchell, sounds good!